piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py

```diff
@@ -1565,7 +1565,9 @@ def cb_sbp_imu_aux(self, msg_raw, **metadata):
         self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale
 
         self.has_imu_scale = True
```

Comment:
Please remove this line.

Reply:
I don't understand which line.
This one?
`self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale`

Reply:
```self.has_imu_scale = True``` is currently set twice. Once in [line 1567](https://github.com/ethz-asl/ethz_piksi_ros/pull/182/files#diff-e9d298b4dd316d49d65e3f18e24f331dR1567), once in [line 1570](https://github.com/ethz-asl/ethz_piksi_ros/pull/182/files#diff-e9d298b4dd316d49d65e3f18e24f331dR1570). Please remove it from line 1567.